modules/gui/src/com/haulmont/cuba/gui/app/core/categories/CategoryBrowser.java

```diff
@@ -134,4 +145,27 @@ public void actionPerform(Component component) {
             }
         }
     }
+
+    protected void initAttrDataTypeColumn() {
+        attributesTable.removeGeneratedColumn("dataType");
+        attributesTable.addGeneratedColumn("dataType", attribute -> {
+            Label dataTypeLabel = componentsFactory.createComponent(Label.class);
```

Comment:
use `new Table.PlainTextCell` instead of Label. `PlainTextCell` has lightweight markup.